### DISCUSSION TITLE
Auto start script

### DISCUSSION THREAD

**Discussion by dshieldspgh** [MLB-LED-Scoreboard/mlb-led-scoreboard#477] [Q&A]
Hi everyone.  Apologies, noob here and having issues with the auto start script.  When I start the scoreboard and check the status, I get a status=200/CHDIR "Failed CHDIR spawning /usr/bin/python3: No such file of directory."
I am running on the 32-bit OS Lite (Debian Bullseye w/ no desktop) and my script is as follows: 

[Unit]
Description=MLB LED Scoreboard
Wants=network-online.target
After=network.target network-online.target

[Service]
Environment="SCOREBOARD_ARGS=--led-rows=32 --led-cols=64 --led-brightness=60 --led-slowdown-gpio=4 --led-gpio-mapping=adafruit-hat-pwm"
WorkingDirectory=/home/pi/mlb-led-scoreboard
ExecStart=/usr/bin/python3 main.py $SCOREBOARD_ARGS
Restart=always
RestartSec=2

[Install]
WantedBy=multi-user.target

Thanks!

**Comment by pjockey** [comment]
You're missing g the start of the command line ,  sudo python3 main.py..........

**Reply by dshieldspgh** [reply]
Thanks for the quick reply.  Adding sudo at the ExecStart did not change the output.  I did change usr to root and changed the status to 203/EXEC (was: 200/CHDIR).  As I understand it, the 203/EXEC is a "file not found" error.  I did confirm the scoreboard service file was in the correct directory (/lib/systemd/system/mlb-led-scoreboard.service).  
![Picture1](https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/assets/117602445/09e227c0-53df-4731-8294-29532275ee92)

**Reply by dshieldspgh** [reply]
fixed the problem.  Didn't have the proper working directory path.

**Comment by pjockey** [comment]
I'm using Supervisor to start and stop the program and this is how my command looks:
```
command=sudo python3 main.py --led-gpio-mapping=adafruit-hat-pwm --led-brightness=60 --led-slowdown-gpio=2 --led-cols=128 --led-rows=64 --led-rgb-sequence=BGR --led-row-addr-type=3
directory=/home/pi/mlb-led-scoreboard
autostart=false
autorestart=true
```

**Reply by jayg7800** [reply]
Thank you, this worked perfectly for me, while using systemctl did not.
